code/modules/mob/living/carbon/emote.dm

```diff
@@ -100,13 +100,39 @@
 		return FALSE
 	return ..()
 
+/datum/emote/living/carbon/mchitter
+	key = "mchitter"
```

Comment:
You can probably just make these ``chitter`` and ``squeak``, we don't have any key collisions as other chitter and squeak are exclusive to lobstrocities and mice